package.json

```diff
@@ -12,7 +12,7 @@
     "bluebird": "^3.5.2",
     "chalk": "2.4.1",
     "commander": "^2.18.0",
-    "debug": "4.0.1",
+    "debug": "4.1.0",
```

Comment:
I can split changes in package.json into a separate PR if someone feels strongly about it, but these are so tiny that I personally don't think that would be worth the hassle.